docs/components/PLACEHOLDERS.md

```diff
@@ -346,3 +397,52 @@ depend on the event itself (Easter or next Summer Holidays for example).
 This means that ``increment`` might have a different value depending on the context of the page which might be tricky to
 maintain and to debug.
 Try to keep it as simple as possible.
+
+### Multiple templates in same placeholder
+
+You can use placeholder actions to target the same placeholderId with different template URLs.
+It groups the rendered templates in the same placeholder, and you can choose the order by using the `priority` attribute
+in the action.
+If not specified, the priority defaults to 0. Then the higher the number, the higher the priority. The final results are
+displayed in descending order of priority.
+The placeholder component waits for all the calls to be resolved (not pending) to display the content.
+The placeholder component ignores a template if the application failed to retrieve it.
+
+
+## Reference CSS classes from an external styling file
+
+You need to reference one or several CSS files from your application in the `cms.json` file:
+
+```json
+{
+  "assetsFolder": "dist/assets",
+  "libraries": [
+    {
+      "npmName": "@o3r/styling"
+    }
+  ],
+  "defaultLanguage": "en-GB",
+  "placeholderCssFiles": [
+    "dist/assets/placeholders/placeholders.css"
+  ]
+}
+```
+
+Those files will be loaded by the CMS to show the placeholder preview.
+Note that you could provide an empty file and update it with the dynamic content mechanism from AEM, to be able to
+reference the new classes afterward.
```

Comment:
I am getting a bit confused by this part. Also, should we mention AEM at all ?

Reply:
This part is just moved from the rules engine documentation and should be covered in the placeholder (component) documentation (refacto).
See #1861